addons/binding/org.openhab.binding.edimax/ESH-INF/i18n/edimax.properties

```diff
@@ -0,0 +1,17 @@
+# binding
```

Comment:
Imho this should not be needed, because all english texts should already be on the specific locations

@kaikreuzer can you confirm this?

Reply:
@fharni Please remove this file